README.md

```diff
@@ -90,6 +90,8 @@ Supported crates such as `cargo-binstall` itself can also be updated with `cargo
   - You can find a full description of errors including exit codes here: <https://docs.rs/binstalk/latest/binstalk/errors/enum.BinstallError.html>
 - Can I use it in CI?
   - Yes! For GitHub Actions, we recommend the excellent [taiki-e/install-action](https://github.com/marketplace/actions/install-development-tools), which has explicit support for selected tools and uses `cargo-binstall` for everything else.
+- Are debug symbols available?
+  - Yes! Extra pre-built packages with a `.full` suffix are available and contain split debuginfo (for Windows and macOS, Linux coming soon), documentation files, and extra binaries like the `detect-wasi` utility.
```

Comment:
I think we should also override `detect-wasi:Cargo.toml:package.binstall` so that it would work as expected?